app/views/dashboard/autoyast.xml.erb

```diff
@@ -10,6 +10,7 @@
         <source><![CDATA[
         #!/bin/sh
         echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        echo "grains_refresh_every: 10" >> /etc/salt/minion.d/master.conf
```

Comment:
The problem with doing it in salt, it that when it gets updated, we need to restart the minion. Doing that in a run will cut off the rest of the highstate / orchestration step currently in progress.